cmd/gitlab.go

```diff
@@ -466,40 +469,39 @@ func (c *Client) findProjectsFromWildcards() error {
 	return nil
 }
 
-func (c *Client) pollProjectsWith(numWorkers int, until <-chan struct{}, projects ...Project) []error {
-	var errs []error
+func (c *Client) pollProjectsWith(numWorkers int, doing func(Project) error, until <-chan struct{}, on ...Project) <-chan error {
 	errorStream := make(chan error)
-	defer close(errorStream)
-	projectsToPoll := make(chan Project, len(projects))
+	projectsToPoll := make(chan Project, len(on))
+	// sync closing the error channel via a waitGroup
+	wg := sync.WaitGroup{}
```

Comment:
```suggestion
	wg := new(sync.WaitGroup)
```

Reply:
`new()`has never been used in the codebase, plus I prefer not to use it at all in any of my projects in favour of `&T{}` or `var X T` for initialisation.
The reason for doing this is having a consistent readability across codebases.
So I'd use `var wg sync.WaitGroup` here, because we adopted it elsewhere in the project.

Reply:
sounds good 👍